Basic-Captions/js/app.js

```diff
@@ -0,0 +1,133 @@
+/* global $ OT SAMPLE_SERVER_BASE_URL */
+
+let apiKey;
+let sessionId;
+let token;
+let publisher;
+let subscriber;
+
+// clears after a set amount of time
+let captionsRemovalTimer;
+
+$(document).ready(function ready() {
+  // Make an Ajax request to get the OpenTok API key, session ID, and token from the server
+  $.get(SAMPLE_SERVER_BASE_URL + '/session', function get(res) {
+    apiKey = res.apiKey;
+    sessionId = res.sessionId;
+    token = res.token;
+
+    initializeSession();
+  });
+});
+
+async function enableTranscription(session) {
+  const { sessionId, token } = session;
+  const url = `${SAMPLE_SERVER_BASE_URL}/captions/start`;
+
+  let captionRes;
+
+  try {
+    captionRes = $.post(url, {
+      sessionId,
+      token,
+    }).done(function response() {
+      console.log('captions started with ID' + captionRes.responseText);
+    });
+  } catch (e) {
+    console.warn(e);
+  }
+}
+
+async function disableTranscription() {
+  const url = `${SAMPLE_SERVER_BASE_URL}/captions/stop`;
+
+  try {
+    $.post(url, {}).done(function response() {
+      console.log('successfully stopped captions');
+    });
+  } catch (e) {
+    console.warn(e);
+  }
+}
+
+
+async function initializeSession() {
+  let session = OT.initSession(apiKey, sessionId);
+
+  // Subscribe to a newly created stream
+  session.on('streamCreated', async function streamCreated(event) {
+    const subscriberOptions = {
+      insertMode: 'append',
+      width: '100%',
+      height: '100%',
+      testNetwork: true,
+    };
+    subscriber = session.subscribe(event.stream, 'subscriber', subscriberOptions, function callback(error) {
+      if (error) {
+        console.log('There was an error publishing: ', error.name, error.message);
+      }
+    });
+
+    // add captions to the subscriber object
+    try {
+      await subscriber.subscribeToCaptions(true);
+    } catch (err) {
+      console.warn(err);
+    }
+
+    subscriber.on('captionReceived', function(event){
+      const captionText = event.caption;
+      const subscriberContainer = OT.subscribers.find().element;
+      const [subscriberWidget] = subscriberContainer.getElementsByClassName('OT_widget-container');
+    
+      const oldCaptionBox = subscriberWidget.querySelector('.caption-box');
+      if (oldCaptionBox) oldCaptionBox.remove();
```

Comment:
Super minor nit pick, but we may want to consider breaking this conditional into multiple lines so that it looks cleaner i.e. 
```
if (oldCaptionBox) {
    oldCaptionBox.remove();
}